src/applications/check-in/components/pages/AppointmentDetails/index.jsx

```diff
@@ -125,12 +135,18 @@ const AppointmentDetails = props => {
                     : t('in-person-appointment')
                 }`}
               </h1>
-              {app === APP_NAMES.PRE_CHECK_IN ? (
-                preCheckInSubTitle
+              {isUpcoming ? (
```

Comment:
Could probably change this to `{!isUpcoming && (...)`

Reply:
I thought about that, but I was like, nah, maybe he wants to add something here in the future

Reply:
I fixed it. I think my brain defaults to this syntax for some reason. But thanks for the benefit of the doubt there Lee!